recipes/duckdb/all/conandata.yml

```diff
@@ -1,4 +1,7 @@
 sources:
+  "0.10.3":
+    url: "https://github.com/duckdb/duckdb/archive/refs/tags/v0.10.3.tar.gz"
```

Comment:
They are offering official source downloads, so please use them instead of the auto-generated tarballs

Reply:
@Croydon 
Are you referring to libduckdb-src.zip?
It contains only three files (duckdb.cpp, duckdb.h, duckdb.hpp) and duckdb.cpp is too large to compile in the CCI ci environment.

Reply:
Ah, my bad, sorry